tests/test_minimizer.py

```diff
@@ -107,7 +107,10 @@ def test_pre_equilibrate_host_pfkfb3(host_name, mol_pair):
         host_config = HostConfig(complex_system, complex_coords, complex_box, num_water_atoms, complex_top)
     x_host, x_box = minimizer.pre_equilibrate_host(mols, host_config, ff)
     assert x_host.shape == host_config.conf.shape
-    assert compute_box_volume(x_box) < compute_box_volume(host_config.box)
+    box_vol_before = compute_box_volume(host_config.box)
```

Comment:
Surprised by this being sensitive to the ligand charges... thanks for cleaning this up.

Reply:
This assertion was False by a very small margin `assert 89.02487850982907 < 88.91724368879996` at https://github.com/proteneer/timemachine/pull/1397/commits/895748d4907f2f536e44657d39ea10edfaa7215c .

(I didn't double-check if the change is still necessary with the slightly different charges at https://github.com/proteneer/timemachine/pull/1397/commits/828a5bdf46e019968a56077eb9efd62f3c65a2da , but would leave the extra wiggle room just in case of future changes.)

Reply:
No need to check, fundamentally there is no guarantee that the box volume is smaller. As long as the box volume is about the same it makes sense to me.